tests/ImageSharp.Benchmarks/ImageSharp.Benchmarks.csproj

```diff
@@ -25,7 +25,9 @@
     <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Condition="'$(OS)' == 'Windows_NT'" />
     <PackageReference Include="Colourful" />
     <PackageReference Include="Pfim" />
-    <PackageReference Include="SharpZipLib"  />
+    <!-- Enable support for using libgdiplus on macOS -->
```

Comment:
This reference should be in the [Directory.Build.targets](https://github.com/SixLabors/ImageSharp/blob/120080b99f3f1be0fc4ecec6906ee683ea999a3d/tests/Directory.Build.targets) file. 

The reference can then be included in the relevant test projects and filtered by OS in a manner similar to the following.
https://github.com/SixLabors/ImageSharp/blob/120080b99f3f1be0fc4ecec6906ee683ea999a3d/tests/ImageSharp.Benchmarks/ImageSharp.Benchmarks.csproj#L25

Reply:
'$(OS)' have only two options: 'Windows_NT' and 'Unix'. So... using `$([MSBuild]::IsOSPlatform('OSX'))` more precise and we do not use non-relevant package in Linux.

Reply:
Ah yes, I forgot this was updated. Yep that's a better filter!